src/SFA.DAS.Campaign.Web/Controllers/EmployerInform/HowDoTheyWorkController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Campaign.Web.Helpers;
+
+namespace SFA.DAS.Campaign.Web.Controllers.EmployerInform
+{
+    public class HowDoTheyWorkController : Controller
+    {
+        private readonly ISessionService _sessionService;
+
+        public HowDoTheyWorkController(ISessionService sessionService)
+        {
+            _sessionService = sessionService;
+        }
+        
+        [HttpGet("employer/how-do-they-work")]
+        public IActionResult Index()
+        {
+            var vm = 
+                _sessionService.Get<LevyOptionViewModel>("LevyOptionViewModel") 
+                ?? new LevyOptionViewModel() {LevyStatus = LevyStatus.NonLevy};
+
+            return View("~/Views/EmployerInform/HowDoTheyWork.cshtml", vm);
+        }
+
+        [HttpPost("employer/how-do-they-work")]
+        public IActionResult Index(LevyOptionViewModel vm)
+        {
+            vm.PreviouslySet = true;
```

Comment:
I don't understand why PreviouslySet would be set to true here, is it appropriate?

Reply:
It's so that next time the VM is retrieved we know that it has been purposefully set by someone and isn't just default values so the UI can be hidden.

Reply:
I can see how the name could be misleading as it's not "previously" to when it's being set here, but it will be "previously" when it's read later.

Reply:
Yeah, its somewhat confusing terminology

Reply:
Renamed the variable to something clearer.